README.md

```diff
@@ -1,17 +1,78 @@
+can still benefit from the above listed features. Here's an example of a very minimally
+abstracted page object, where we're using a page object assertion (`title_should_be`) 
+outside of the Robot framework. `mytest.py`  
+
+    from robotpageobjects import Page
+    from unittest import TestCase
 
-The main point of using page objects is to factor out page implementation details (locators, UI details etc.) from the actual test suites. This makes the tests read more about the services a page offers and what's being tested instead of the internals of the page. It also makes your tests much more maintainable. For example, if a developer changes an element ID, you only need make that change once--in the appropriate page object.
 
-Each page object is simply a Robot library that inherits from this package's base `Page` class. These library classes can work independently of Robot
-Framework, even though they ultimately inherit from Robot Framework's Selenium2Library. This  allows you to encapsulate page logic Robot libraries, but use those libraries in any testing framework, including 
-Python [unittest](https://docs.python.org/2/library/unittest.html) test cases.
+    class MyPage(Page):
+        uri = "/some/path"
+        selectors = {
+            "the portlet": "xpath://some/complicated/xpath"
+        }
 
+    class MyTestCase(TestCase):
+        def setUp(self):
+            self.page = Page()
+            self.page.open()
 
-## Installing
+        def tearDown(self):
+            self.page.close()
 
-    $ pip install robotframework-pageobjects
+        def test_title(self):
+            self.page.title_should_be("My Page")
+
+        def test_portlet_renders(self):
+            self.page.element_should_be_visible("the portlet")
+
+
+We could run this test on the default browser ([PhantomJs](http://phantomjs.org/)) like this:
+
+    $ export PO_BASEURL=http://qa.mydomain.com 
+    $ export PO_SELENIUM_SPEED=1 # Slow the whole test down for debugging
+    $ python mytest.py
+
+Notice, we did not factor out all page implementation details from the test itself. But
+we still can leverage many of the package's features in our tests. If we need to model the page
+further, nothing stops us from doing so in the future. 
+We'll learn how to do this in a bit.
+
+## More on page objects
+The main point of using page objects is to factor out page implementation details (element locators, UI details etc.) from the actual test suites. This makes the tests read more about the services a page offers and what's being tested instead of the internals of the page. It also makes your tests much more maintainable. For example, if a developer changes an element ID, you only need make that change once--in the appropriate page object.
+
+## How it works
+Each page object you create is simply a Robot library that inherits from this package's base `Page` class. Since these classes are *plain old Python classes* they can work independently of Robot
```

Comment:
To maintain the framework-agnostic image, I'd change "a Robot library" to "an object" in the first part of this sentence and then mention that in the context of robot, it becomes a "Robot library"